nixos/modules/services/misc/etcd.nix

```diff
@@ -184,7 +190,7 @@ in {
       };
     };
 
-    environment.systemPackages = [ pkgs.etcd ];
+    environment.systemPackages = [ cfg.package ];
```

Comment:
This needs to be updated in the `serviceConfig` as well.

Reply:
Done.